system-tests/__snapshots__/non_root_read_only_fs_spec.ts.js

```diff
@@ -16,14 +16,14 @@ If you don't require screenshots or videos to be stored you can safely ignore th
   ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
   │ Cypress:    1.2.3                                                                              │
   │ Browser:    FooBrowser 88                                                                      │
-  │ Specs:      1 found (spec.js)                                                                  │
-  │ Searched:   cypress/integration/spec.js                                                        │
+  │ Specs:      1 found (spec.cy.js)                                                               │
+  │ Searched:   cypress/e2e/spec.cy.js                                                             │
   └────────────────────────────────────────────────────────────────────────────────────────────────┘
```

Comment:
These snapshot updates look good but should these snapshot files been renamed from `_spec.ts` to `.cy.ts`

Reply:
They have been 👍